app/build.gradle

```diff
@@ -55,8 +55,8 @@ dependencies {
     implementation "com.google.firebase:firebase-messaging:$rootProject.firebaseVersion"
     implementation "com.android.support.constraint:constraint-layout:$rootProject.constraintLayoutVersion"
     testImplementation "junit:junit:$rootProject.junitVersion"
-    androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+    androidTestImplementation 'com.android.support.test:runner:1.0.1'
```

Comment:
Why not 1.0.2?

Reply:
I previously set it to 1.02 but @sagarsrao said I should not modify the build.gradle file so I changed them back to 1.0.1

Reply:
Okay

Reply:
What should I do?
Should I change it back to 1.0.2?
Also should I push the updated UI to the branch with **Google Udacity Scholars** text and India Logo?

Reply:
@YuganshT79 Yes change it to 1.0.2. Just do a git pull, or compare your recent code with the latest commit in the branch. This way there won't be any conflicts.

Reply:
Ok, will review the changes and Push the code, and what about the updated UI code ?

Reply:
What exactly do you mean by updated UI code? If there's some updates in the profile page code (xml file) from your end, include that as well in the commit.

Reply:
Sure, will do 👍